### PR TITLE
Add OrderByClause to SelectBuilder

### PR DESCRIPTION
**Pull request by xdefrag**
Just bumped into this issue https://github.com/Masterminds/squirrel/issues/139 and implemented OrderByClause.

### CODE REVIEW

**Comment by xdefrag** [comment]
@lann can you review this? If PR need some changes, I will do them shortly.

File: select.go [hunk 1]
```diff
@@ -129,9 +129,9 @@ func (d *selectData) toSql() (sqlStr string, args []interface{}, err error) {
 		}
 	}
 
-	if len(d.OrderBys) > 0 {
+	if len(d.OrderByParts) > 0 {
 		sql.WriteString(" ORDER BY ")
-		sql.WriteString(strings.Join(d.OrderBys, ", "))
+		args, err = appendToSql(d.OrderByParts, sql, ", ", args)
```
Comment (lann) [inline select.go]: Need to check and return err here.

Comment (xdefrag) [reply select.go]: Done

**Comment by lann** [comment]
Thanks!

**Pull request merged**